### ISSUE TITLE
logger object does not have a debug function ?

### ISSUE THREAD

**Issue by mdenty** [madchicken/homebridge-zigbee-nt#221]
After having this consistent error with my LIDL bulb
```
[Fri Dec 17 2021] [ZigBee-UI] WEB UI Listening on port 9000
[12/17/2021, 5:35:00 PM] [ZigBee] Device 0x84fd27fffec7836f successfully configured on attempt 1!
[12/17/2021, 5:35:14 PM] TypeError: Cannot read properties of undefined (reading 'debug')
    at findColorTempRange (/usr/lib/node_modules/homebridge-zigbee-nt/node_modules/zigbee-herdsman-converters/lib/light.js:69:16)
    at Object.syncColorState (/usr/lib/node_modules/homebridge-zigbee-nt/node_modules/zigbee-herdsman-converters/lib/color.js:651:42)
    at Object.convert (/usr/lib/node_modules/homebridge-zigbee-nt/node_modules/zigbee-herdsman-converters/converters/fromZigbee.js:599:51)
    at /usr/lib/node_modules/homebridge-zigbee-nt/src/zigbee/zig-bee-client.ts:127:37
    at Array.forEach (<anonymous>)
    at ZigBeeClient.decodeMessage (/usr/lib/node_modules/homebridge-zigbee-nt/src/zigbee/zig-bee-client.ts:123:18)
    at ZigbeeNTHomebridgePlatform.handleZigBeeMessage (/usr/lib/node_modules/homebridge-zigbee-nt/src/platform.ts:414:19)
    at Controller.<anonymous> (/usr/lib/node_modules/homebridge-zigbee-nt/src/platform.ts:124:71)
    at Controller.emit (node:events:390:28)
    at Controller.onZclOrRawData (/usr/lib/node_modules/homebridge-zigbee-nt/node_modules/zigbee-herdsman/src/controller/controller.ts:686:18)
```
I've commented out the `logger.debug` statement at line 69 and now it seems to work fine. 

I am a total newbie with Javascript, but it seems that the `logger` object is broken.

**Comment by madchicken** [comment]
@mdenty it seems a bug in the herdsman lib (a library I use as a dependency).
I just released a new version using an updated version of that library. Let's see if this is still happening. Feel free to reopen this issue if it does.

**Comment by mdenty** [comment]
@madchicken Yes it still occurs.
```
[1/16/2022, 12:21:50 PM] [ZigBee] No need to configure device 0x84fd27fffec7836f
[1/16/2022, 12:21:51 PM] TypeError: Cannot read properties of undefined (reading 'debug')
    at findColorTempRange (/usr/lib/node_modules/homebridge-zigbee-nt/node_modules/zigbee-herdsman-converters/lib/light.js:69:16)
    at Object.syncColorState (/usr/lib/node_modules/homebridge-zigbee-nt/node_modules/zigbee-herdsman-converters/lib/color.js:651:42)
    at Object.convert (/usr/lib/node_modules/homebridge-zigbee-nt/node_modules/zigbee-herdsman-converters/converters/fromZigbee.js:603:51)
    at /usr/lib/node_modules/homebridge-zigbee-nt/src/zigbee/zig-bee-client.ts:139:37
    at Array.forEach (<anonymous>)
    at ZigBeeClient.decodeMessage (/usr/lib/node_modules/homebridge-zigbee-nt/src/zigbee/zig-bee-client.ts:135:18)
    at ZigbeeNTHomebridgePlatform.handleZigBeeMessage (/usr/lib/node_modules/homebridge-zigbee-nt/src/platform.ts:396:19)
    at Controller.<anonymous> (/usr/lib/node_modules/homebridge-zigbee-nt/src/platform.ts:124:71)
    at Controller.emit (node:events:390:28)
    at Controller.onZclOrRawData (/usr/lib/node_modules/homebridge-zigbee-nt/node_modules/zigbee-herdsman/src/controller/controller.ts:661:18)
[1/16/2022, 12:21:51 PM] Got SIGTERM, shutting down Homebridge...
``` 
I'm a total noob with node js but when I go to : `/usr/lib/node_modules/homebridge-zigbee-nt`
and type : `npm list`
```
npm list
homebridge-zigbee-nt@2.5.6 /usr/lib/node_modules/homebridge-zigbee-nt
├── UNMET DEPENDENCY @emotion/eslint-plugin@^11.2.0
├── UNMET DEPENDENCY @nrwl/eslint-plugin-nx@^11.4.0
├── UNMET DEPENDENCY @trongnd/ts-nodemon@^1.0.3
├── UNMET DEPENDENCY @types/async-retry@^1.4.2
├── UNMET DEPENDENCY @types/express@^4.17.9
├── UNMET DEPENDENCY @types/fs-extra@7.0.0
├── UNMET DEPENDENCY @types/jest@^26.0.20
├── UNMET DEPENDENCY @types/lodash@^4.14.149
├── UNMET DEPENDENCY @types/nanoid@^2.1.0
├── UNMET DEPENDENCY @types/node@^11.15.3
├── UNMET DEPENDENCY @types/react-dom@17.0.0
├── UNMET DEPENDENCY @types/react-router-dom@^5.1.6
├── UNMET DEPENDENCY @types/react-router@^5.1.8
├── @types/react@16.14.21 invalid: "17.0.0" from the root project
├── UNMET DEPENDENCY @types/serialport@^8.0.1
├── UNMET DEPENDENCY @types/serve-static@^1.13.9
├── UNMET DEPENDENCY @types/ws@^7.4.0
├── UNMET DEPENDENCY @typescript-eslint/eslint-plugin@^4.15.2
├── UNMET DEPENDENCY @typescript-eslint/parser@^4.15.2
├── async-retry@1.3.1
├── body-parser@1.19.0
├── UNMET DEPENDENCY concurrently@^5.3.0
├── dayjs@1.10.5
├── emotion@11.0.0
├── UNMET DEPENDENCY eslint-config-prettier@^8.1.0
├── UNMET DEPENDENCY eslint-plugin-jest@^24.1.5
├── UNMET DEPENDENCY eslint-plugin-react-hooks@^4.2.0
├── UNMET DEPENDENCY eslint-plugin-react@^7.22.0
├── UNMET DEPENDENCY eslint@^7.21.0
├── evergreen-ui@6.6.3
├── express@4.17.1
├── fakegato-history@0.6.1
├── UNMET DEPENDENCY faker@^5.4.0
├── UNMET DEPENDENCY homebridge@^1.3.8
├── UNMET DEPENDENCY http-proxy-middleware@^1.0.6
├── http2@3.3.7
├── UNMET DEPENDENCY husky@^7.0.4
├── UNMET DEPENDENCY jest@^27.4.4
├── js-big-decimal@1.2.4
├── lodash@4.17.21
├── UNMET DEPENDENCY nock@^12.0.2
├── node-forge@1.2.1 invalid: "^0.10.0" from the root project
├── UNMET DEPENDENCY parcel@^2.0.1
├── UNMET DEPENDENCY prettier@^2.5.1
├── UNMET DEPENDENCY pretty-quick@^3.1.2
├── react-dom@17.0.0
├── react-is@17.0.0
├── react-json-view@1.21.3
├── react-query@3.17.2
├── react-router-dom@5.2.0
├── react-router@5.2.0
├── react@17.0.0
├── rimraf@3.0.2
├── serialport@9.0.6
├── serve-static@1.14.1
├── UNMET DEPENDENCY ts-jest@^27.1.2
├── UNMET DEPENDENCY ts-node@^9.1.1
├── UNMET DEPENDENCY typescript@^4.2.2
├── winston@3.3.3
├── ws@7.5.6
├── UNMET DEPENDENCY yarn-audit-fix@^6.3.4
├── zigbee-herdsman-converters@14.0.366
└── zigbee-herdsman@0.13.188
```
Is this normal ?


For now, I've commented out the line 69 in `/usr/lib/node_modules/homebridge-zigbee-nt/node_modules/zigbee-herdsman-converters/lib/light.js`: it makes it work.

**Comment by madchicken** [comment]
@mdenty I think I found the issue. I'll reopen this, please wait for the next version
